.github/workflows/dismiss-stale-pr-reviews.yml

```diff
@@ -18,7 +18,27 @@ jobs:
   dismiss_stale_approvals:
     runs-on: ubuntu-latest
     steps:
+      - name: Check Core Team Membership
+        env:
+          ACTOR: ${{ github.actor }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          TEAM_ID: '11759072'
+        run: |
+          curl -X GET \
+            https://api.github.com/teams/${TEAM_ID}/memberships/${ACTOR} \
+              -H "Authorization: Bearer ${GITHUB_TOKEN}" \
+              -H "Accept: application/vnd.github+json" \
+              -o response.json
```

Comment:
I have absolutely no idea if this actually works.

Reply:
I think you need to pass `--fail` to curl to make it exit with a non-zero status code on the 404
https://superuser.com/a/1626376